pkg/collector/collector.go

```diff
@@ -69,3 +76,26 @@ func (b *BaseCollector) Export() error {
 
 	return nil
 }
+
+// Helper function to collect output of a given kubectl command to a file. Returns kubectl's stderr output if stdout output is empty.
+func (b *BaseCollector) collectResourceToFileUsingKubectl(rootPath, fileName string, kubeCmds []string) error {
```

Comment:
can this name be simplified to something shorter like `collectKubeResourceToFile` ?